### PR TITLE
always like future time stamps

### PR DESCRIPTION
**Pull request by apenzk**
# Description of change
Always like future transactions

## Type of change
set like to all future messages. This is a temporary solution until the booker/scheduler is updated.

### CODE REVIEW

File: packages/consensus/fcob/timestamp.go [hunk 1]
```diff
@@ -18,13 +18,23 @@ var (
 // region TimestampQuality /////////////////////////////////////////////////////////////////////////////////////////////
 
 // TimestampQuality returns the TimestampOpinion based on the given times (e.g., arrival and current).
-func TimestampQuality(messageID tangle.MessageID, target, current time.Time) (timestampOpinion *TimestampOpinion) {
-	diff := abs(current.Sub(target))
-
+func TimestampQuality(messageID tangle.MessageID, target, current time.Time) (timestampOpinion *TimestampOpinion, err error) {
```
Comment (capossele) [inline packages/consensus/fcob/timestamp.go:21]: we do not use the returned error so let's remove it

File: packages/tangle/scheduler.go [hunk 2]
```diff
@@ -98,6 +98,10 @@ func (s *Scheduler) scheduleMessage(messageID MessageID) {
 		return
 	}
 
+	// if !s.isInTheFuture(messageID) {
+	// 	return
+	// }
+
```
Comment (capossele) [inline packages/tangle/scheduler.go:104]:
```suggestion
```

File: packages/tangle/scheduler.go [hunk 3]
```diff
@@ -127,6 +131,20 @@ func (s *Scheduler) parentsBooked(messageID MessageID) (parentsBooked bool) {
...
+// 		current := clock.SyncedTime()
+// 		// current := time.Now()
+// 		if current.Sub(message.issuingTime) < 0 {
+// 			isInTheFuture = true
+// 		}
+// 	})
+
+// 	return
+// }
+
```
Comment (capossele) [inline packages/tangle/scheduler.go:147]:
```suggestion
```

**Review by capossele — CHANGES_REQUESTED** [review]

**Comment by apenzk** [comment]
Make TimestampQuality assymetric

**Pull request merged**